napari_btrack/widgets/_hypothesis.py

```diff
@@ -0,0 +1,217 @@
+            options={"tooltip": tooltip},
+        )
+        hypotheses_widgets.append(widget)
+
+    # P_FP is always required
+    P_FP_hypothesis = hypotheses_widgets[0]
+    P_FP_hypothesis.enabled = False
+
+    # P_merge should be disabled by default
+    P_merge_hypothesis = hypotheses_widgets[-1]
+    P_merge_hypothesis.value = False
+
+    return hypotheses_widgets
+
+
+def _create_scaling_factor_widgets():
+    """Create widgets for setting the scaling factors of the HypothesisModel"""
+
+    values = [5.0, 3.0, 10.0, 50.0]
+    names = [
+        "lambda_time",
+        "lambda_dist",
+        "lambda_link",
+        "lambda_branch",
+    ]
+    labels = [
+        "λ time",
+        "λ distance",
+        "λ linking",
+        "λ branching",
+    ]
+    tooltips = [
+        "Scaling factor for the influence of time when determining initialization or termination hypotheses.",  # noqa: E501
+        "Scaling factor for the influence of distance at the border when determining initialization or termination hypotheses.",  # noqa: E501
+        "Scaling factor for the influence of track-to-track distance on linking probability.",  # noqa: E501
+        "Scaling factor for the influence of cell state and position on division (mitosis/branching) probability.",  # noqa: E501
+    ]
+
+    scaling_factor_widgets = []
+    for value, name, label, tooltip in zip(values, names, labels, tooltips):
+        widget = magicgui.widgets.create_widget(
+            value=value,
+            name=name,
+            label=label,
+            widget_type="FloatSpinBox",
+            options={"tooltip": tooltip},
+        )
+        scaling_factor_widgets.append(widget)
+
+    return scaling_factor_widgets
+
+
+def _create_threshold_widgets():
+    """Create widgets for setting thresholds for the HypothesisModel"""
+
+    tooltip = (
+        "A threshold distance from the edge of the FOV to add an "
+        "initialization or termination hypothesis."
+    )
+    distance_threshold = magicgui.widgets.create_widget(
+        value=20.0,
+        name="theta_dist",
+        label="distance threshold",
+        widget_type="FloatSpinBox",
+        options={"tooltip": tooltip},
+    )
+
+    tooltip = (
+        "A threshold time from the beginning or end of movie to add "
+        "an initialization or termination hypothesis."
+    )
+    time_threshold = magicgui.widgets.create_widget(
+        value=5.0,
+        name="theta_time",
+        label="time threshold",
+        widget_type="FloatSpinBox",
+        options={"tooltip": tooltip},
+    )
+
+    tooltip = (
+        "Number of apoptotic detections to be considered a genuine event.\n"
+        "Detections are counted consecutively from the back of the track"
+    )
+    apoptosis_threshold = magicgui.widgets.create_widget(
+        value=5,
+        name="apop_thresh",
+        label="apoptosis threshold",
+        widget_type="SpinBox",
+        options={"tooltip": tooltip},
+    )
```

Comment:
I took the defaults from the `cell_config.json`. We could pass a dictionary of default values into each function, but I'd rather not pass in `TrackerConfig` itself to the functions, because if the structure of `TrackerConfig` changes we have to update all of these calls to `widget_creation` to handle the new `TrackerConfig`

Reply:
I've added a couple of lines to [explicitly set the default values](https://github.com/lowe-lab-ucl/napari-btrack/blob/fix/remove-modes/napari_btrack/main.py#L54-L58) to be those of the cell_config. This means if the default values of `cell_config` ever change, we won't need to update the defaults for the gui as it's done automatically